test/hotspot/jtreg/runtime/NMT/MallocRoundingReportTest.java

```diff
@@ -57,15 +57,15 @@ public static void main(String args[]) throws Exception {
                 // NMT does not track memory allocations less than 1KB, and rounds to the nearest KB
                 NMTTestUtils.runJcmdSummaryReportAndCheckOutput(
                         "Test (reserved=" + numKB + "KB, committed=" + numKB + "KB)",
-                        "(malloc=" + numKB + "KB #1) (at peak)"
+                        "(malloc=" + numKB + "KB type=12 #1) (at peak)"
```

Comment:
Again, this is wrong. The memory had not been reserved for NMT usage, so it should not be tagged with mtNMT (12)

Reply:
Looking into it...